### PR TITLE
New package: razergenie-0.6

### PR DESCRIPTION
**Pull request by z3ntu**
~~Requires openrazer-meta, a package that will be added in a later pull request. So I'd like it just to be reviewed for now :)~~

### CODE REVIEW

**Comment by Johnnynator** [comment]
I can not say much to it atm since obvious reasons. Just that all outcommented or empty variables should be removed.

> Requires openrazer-meta, a package that will be added in a later pull request
You should add this to this PR too.

**Comment by z3ntu** [comment]
Added openrazer-meta plus dependencies.

**Comment by Johnnynator** [comment]
You should take a look what xlint gives/would give you for errors. You have a few small mistakes, like missing maintainer, empty hostmakedepends, unneeded trailing dot in descriptions.

File: srcpkgs/openrazer-meta/template [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		vmove usr/bin
+		vmove $py3_sitelib/openrazer_daemon
+		vmove $py3_sitelib/openrazer_daemon-*.egg-info
+		vmove usr/share
+	}
+}
+
+openrazer-driver-dkms_package() {
+	short_desc="Kernel driver for Razer devices (DKMS-variant)"
+	depends="dkms"
```
Comment (Johnnynator) [inline srcpkgs/openrazer-meta/template]: You probably should also the dkms trigger and specify the dkms_modules name.
```
triggers="dkms"
dkms_modues="openrazer-driver 1.0.0"
```

**Comment by z3ntu** [comment]
I've fixed all errors by xlint and added the dkms trigger (didn't know this existed, it isn't mentioned in the manual.md file)

**Comment by pullmoll** [comment]
Please squash your commits so that for each new/modified package there is just one commit.

**Comment by z3ntu** [comment]
Yes I will do that when the templates are fine. Any other comments about the templates?
There's also some way missing of auto-starting the `openrazer-daemon`, so integration with `sv up` is still a TODO.

**Comment by Johnnynator** [comment]
You can try to fix cross compilation. Otherwise I think they are fine

**Comment by z3ntu** [comment]
@osum4est Can you try this PR? Everything **should** work now.

File: srcpkgs/openrazer-meta/template [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		vmove usr/lib/udev
+		vmove usr/src
+	}
+}
+
+python3-openrazer_package() {
+	short_desc="Python library for accessing the Razer daemon from Python"
+	depends="python3 python3-dbus python3-numpy python3-gobject openrazer-daemon"
+	triggers="dkms"
+	dkms_modules="openrazer-driver 1.0.0"
```
Comment (Johnnynator) [inline srcpkgs/openrazer-meta/template]: You should move these two into the dkms subpackage

File: srcpkgs/razergenie/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'razergenie'
+pkgname=razergenie
+version=0.6
+revision=1
+wrksrc=RazerGenie-${version}
+build_style=meson
+makedepends="pkg-config qt5-devel"
```
Comment (Johnnynator) [inline srcpkgs/razergenie/template]: pkg-config should be in hostmakedepends

File: srcpkgs/openrazer-meta/template (hunk 2 above)
Comment (z3ntu) [reply srcpkgs/openrazer-meta/template]: idk how that landed in there ^^ it's fixed now

**Comment by z3ntu** [comment]
Cross compilation was tested for: aarch64 & aarch64-musl. I hope everything is alright now :)

**Comment by z3ntu** [comment]
bump

**Pull request closed without merging**